### ISSUE TITLE
4.16.0.0 UI BUG: SSH Key Pair Navigation

### ISSUE THREAD

**Issue by aronwagner23** [apache/cloudstack#5930]
##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
UI
~~~

##### CLOUDSTACK VERSION

~~~
4.16.0.0
~~~

##### SUMMARY

Our region has multiple SSH Key Pairs but in the UI when clicking on any of them it always routes to the same one key.

##### STEPS TO REPRODUCE

~~~
compute -> ssh key pairs -> click different names in the list -> observe all going to the same single ssh key
~~~

##### EXPECTED RESULTS

~~~
Being able to navigate to multiple ssh key pairs
~~~

##### ACTUAL RESULTS

~~~
Only navigates to the same ssh key pair no matter which one I click
~~~

**Comment by nvazquez** [comment]
I got the same error, always getting to the last created key, even though the UUID is correctly set in the URL

**Comment by sureshanaparti** [comment]
Fixed in #5931
